app/models/job_execution.rb

```diff
@@ -109,6 +110,7 @@ def execute!(dir)
       "export REVISION=#{@reference.shellescape}",
       "export TAG=#{(@job.tag || @job.commit).to_s.shellescape}",
       "export CACHE_DIR=#{artifact_cache_dir}",
+      "export DEPLOY_GROUPS='#{@stage.deploy_groups.map(&:name).join(' ') if @stage}'",
```

Comment:
probably also want to shell escape these. Not sure what would happen if they had a space or single quote